StoreManagementService/src/PBJ.StoreManagementService.Api/Controllers/PostController.cs

```diff
@@ -15,18 +16,20 @@ public PostController(IPostService postService)
             _postService = postService;
         }
 
-        [HttpGet, Route("{amount}")]
-        public async Task<ActionResult> GetAmountAsync(int amount)
+        [HttpGet, Route("paginated")]
+        public async Task<ActionResult> GetPaginatedAsync([FromQuery] PaginationRequestModel requestModel)
         {
-            var result = await _postService.GetAmountAsync(amount);
+            var result = await _postService
+                .GetPaginatedAsync(requestModel.Page, requestModel.Take);
 
             return Ok(result);
         }
 
-        [HttpGet, Route("user")]
-        public async Task<ActionResult> GetUserPostsAsync(int userId, int amount)
+        [HttpGet, Route("userId")]
+        public async Task<ActionResult> GetByPostId(int userId, [FromQuery] PaginationRequestModel requestModel)
         {
```

Comment:
Something wrong with namings, please fix it. It named as **GetByPostId** but nelow you use **GetByUserId**